w3af/plugins/crawl/find_backdoors/web_shells.txt

```diff
@@ -178,3 +178,721 @@ nc.exe
 netcat.exe
 socat.exe
 cmd.pl
+
```

Comment:
Not 100% sure about this, but how are empty lines in this file handled by the plugin? Same with the previous file.

Reply:
Thanks to `line.strip()` empty lines are ignored. 
https://github.com/andresriancho/w3af/blob/193738bf46dd4d9476e6f632df4204ddf71227fb/w3af/plugins/crawl/find_backdoors.py#L103
```
if not line:
    continue
```
https://github.com/andresriancho/w3af/blob/193738bf46dd4d9476e6f632df4204ddf71227fb/w3af/plugins/crawl/find_backdoors.py#L108